doc/source/basics.rst

```diff
@@ -793,8 +793,12 @@ The :meth:`~DataFrame.apply` method will also dispatch on a string method name.
    df.apply('mean')
    df.apply('mean', axis=1)
 
-Depending on the return type of the function passed to :meth:`~DataFrame.apply`,
-the result will either be of lower dimension or the same dimension.
+The return type of the function passed to :meth:`~DataFrame.apply` affects the
```

Comment:
return type -> default return type (and maybe after the two bullet points mention that it can be controlled with the `result_type` keyword (without going into details) ?)